tao_compiler/mlir/xla/ral/context/base/base_context.h

```diff
@@ -75,6 +76,18 @@ class InternalAllocator : public Allocator {
   std::map<buffer_t, size_t> allocated_buffers_;
 };
 
+class TFAllocatorWrapper : public Allocator {
+ public:
+  TFAllocatorWrapper(tensorflow::Allocator* allocator)
+      : allocator_(std::move(allocator)){};
+  ~TFAllocatorWrapper(){};
+  buffer_t alloc(size_t bytes);
+  void dealloc(buffer_t buffer);
+
+ private:
+  std::unique_ptr<tensorflow::Allocator> allocator_;
```

Comment:
we should not expose any TensorFlow symbol in `base_context.h`  since it will be included by other users (e.g. torch blade ) outside TF.

Reply:
Thanks, will update this later.